lib/iota.js

```diff
@@ -42,10 +48,10 @@ IOTA.prototype.setSettings = function(settings) {
 
 
 /**
-*   Change the Node the user connects to
+*   Change the Node the user connects to.
 *
 *   @method changeNode
-*   @param {Object} settings
+*   @param {Object} settings See `setSettings` method for properties.
 **/
 IOTA.prototype.changeNode = function(settings) {
     this.setSettings(settings);
```

Comment:
If the only thing that `changeNode` does is call `setSettings`, could we deprecate it?